### PR TITLE
Show welcome message in success animation ✨

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Show welcome message after the success animation.

Show message also when the user has disabled autoplaying videos.

Check out the preview deployment to see it in action! https://hedvig-dot-com-git-grw-2336-featuresuccess-welcom-ce7abe-hedvig.vercel.app/se/confirmation/6a943839-4f33-40b5-b769-5891f4bd1236

[Untitled.mov](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/OgXegTwxM9IeuXHZyw5h/f5e8d6b4-bf69-43e4-ac00-b4aa71efb186/Untitled.mov)

## Justify why they are needed

Design update.

Hopefully the state reducer makes things easier to understand now!

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1858** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1858" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1858?utm_source=stack-comment).

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/components/ConfirmationPage/SuccessAnimation/SuccessAnimation.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        <AnimationWrapper key="wrapper" {...WRAPPER_ANIMATION}>
+          <AnimatePresence>
+            {state === 'video' && (
+              <StyledVideo
+                ref={videoRef}
+                muted={true}
+                autoPlay={true}
+                playsInline={true}
+                preload="auto"
+                onEnded={completeAnimation}
```
Comment (gustaveen) [inline apps/store/src/components/ConfirmationPage/SuccessAnimation/SuccessAnimation.tsx:32]: Nice, didn't know video had an `onEnded` event 🙌

**Review by gustaveen — APPROVED** [review]
Very slick! 🎉

**Pull request merged**